tmt/checks/dmesg.py

```diff
@@ -187,7 +187,10 @@ def before_test(
             environment: Optional[tmt.utils.Environment] = None,
             logger: tmt.log.Logger) -> list[CheckResult]:
         if not invocation.guest.facts.has_capability(GuestCapability.SYSLOG_ACTION_READ_ALL):
-            return [CheckResult(name='dmesg', result=ResultOutcome.SKIP)]
+            return [CheckResult(
+                name='dmesg',
+                result=ResultOutcome.SKIP,
+                note=['fetching of buffer content was not allowed'])]
```

Comment:
I am wondering, would it not make more sense to link from the note to docs giving more explanation here?
As a user I will have no godd idea what this actually means. Docs could explain more?

Reply:
I agree, it would be nice. I will try tonimprove it - check out my PR about “hints”, I tried to improve this kind of messages, I’d say it’s related, and with `tmt about` I’d like to build a corpus of hints and helpful messages, exceptions, etc.